Makefile.PL

```diff
@@ -1,8 +1,8 @@
 use ExtUtils::MakeMaker;
 WriteMakefile(
 		NAME	=> 'Tie::Scalar::Decay',
+		LICENSE	=> 'perl_5',
     META_MERGE => {
-        license => 'open_source',
         resources => {
```

Comment:
I'm pretty sure that changing the structure like this will break some old versions of EU::MM and so prevent the module from building.

Reply:
Any suggestion to figure out that? It would be nice to know :-)

Reply:
perlbrew install 5.N.last for all even N >= 8 and see what happens :-)

Fortunately I already have all those perls built so I'll try it myself.

Reply:
Tested all the way down to perl 5.6.2, and the worst it does is spit warnings from Makefile.PL.